readme.md

```diff
@@ -1,3 +1,21 @@
 # Design System Team Labels
 
 We use this repo to synchronise the labels we use across our repositories.
+
+It uses [github-label-sync](https://github.com/Financial-Times/github-label-sync).
+
+## Updating labels
```

Comment:
I'm assuming that we're not concerned about being able to delete labels in the future, just adding and modifying them? Maybe the README could mention that labels need to be deleted manually from the repos, if needed (removing `--allow-added-labels` would do it as well but probably not what we want to do in this context).